### DISCUSSION TITLE
I get an error when testing in my Github Repository

### DISCUSSION THREAD

**Discussion by byk1lla** [community/community#128781] [Actions]
### Select Topic Area

Product Feedback

### Body

Hello,

I've been getting notifications for a few days now, but I don't know what to do to fix it, I don't know if it's a general error or if it's something wrong with me.

I have shown the error below

![image](https://github.com/community/community/assets/126905649/465873ad-4441-4bd5-ade8-00c1789f4202)

**Comment by cornel-pe** [comment]
try to update your Node version and Php version.
